build-deb-package

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file BOOST_LICENSE_1.0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
I wonder if we could just drop this and implicitly assume its licence to match that of the upstream project (Apache v2.0: https://github.com/sociomantic-tsunami/mxnet/blob/master/LICENSE)